src/platform/plugins/shared/visualizations/common/utils/prepare_log_table.ts

```diff
@@ -72,6 +96,9 @@ export const prepareLogTable = (
           meta: {
             ...column.meta,
             dimensionName: getDimensionName(column, columnIndex, dimensions),
+            ...(isLayerDimensions(dimensions)
```

Comment:
nit: this doesn't need to be re-evaluated for every column. It can evaluated outside the `map` function and just executed once.

See here: https://github.com/Zacqary/kibana/compare/208854-rule-from-dash-vis...dej611:pr/217719#diff-32a16bd0c9c54ee3edbac8378c759e9011016fabc1611db8cf69e7eb070536a0